tests/end_to_end/test_tables.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+import difflib
+import os
+from pathlib import Path
+
+import yaml
+try:
+    from yaml import CLoader as Loader
+except ImportError:
+    from yaml import Loader
+
+import n2y
+from n2y.main import main
+
+NOTION_ACCESS_TOKEN = os.getenv("NOTION_ACCESS_TOKEN") or 'secret_lylx4iL5awveY3re6opuvSQqM6sMRu572TowhfzPy5r'
+
+def test_simple_table(monkeypatch, request, tmp_path):
+    """
+    This will pass only after a workaround for Markdown's table spec. The dominant dialect of Markdown requires a
+    header row to designate a table, whereas others do not.
+
+    Relies on https://www.notion.so/Simple-Tables-9b1dd705f61647b6a10032ec7671402f?pvs=4
+    """
+    object_id = "9b1dd705f61647b6a10032ec7671402f"
+
+    config = {
+        "exports": [
+            {
+                "id": object_id,
+                "node_type": "page",
+                "output": f"{request.node.name}-output",
+                "pandoc_format": "gfm",
```

Comment:
This format is key. gfm=GitHub-flavored Markdown. We might get better use from other dialects of Markdown.